src/components/dashboard/appShell/header.tsx

```diff
@@ -99,6 +105,21 @@ export const DashboardHeader: FC<DashboardProps> = () => {
               {navLink.name}
             </NavItem>
           ))}
+          <Button
+            isLoading={logoutLoading}
+            leftIcon={<FaSignOutAlt />}
+            colorScheme='red'
+            variant='ghost'
+            size={'lg'}
+            onClick={logout}
+            pl={'15px'}
+            isActive={false}
+            loadingText={'Logging out'}
+            spinnerPlacement={'start'}
+            fontWeight={'normal'}
+          >
+            Logout
+          </Button>
```

Comment:
This button is not full width of the container, please adjust the width.

Reply:
Done